apps/parse/test/parse/alerts_test.exs

```diff
@@ -408,6 +436,92 @@ defmodule Parse.AlertsTest do
       assert [%Alert{description: "Good morning"}] = parse_json(map)
     end
 
+    test "returns english image url if available" do
+      url =
+        "https://dev-mbta.pantheonsite.io/sites/default/files/styles/max_2600x2600/public/media/2023-09/QuincyCenter-Braintree-EA.png"
+
+      map = %{
+        "timestamp" => "1496832813",
+        "alerts" => [
+          %{
+            "active_period" => [],
+            "alert_lifecycle" => "NEW",
+            "cause" => "UNKNOWN_CAUSE",
+            "created_timestamp" => 1_494_947_991,
+            "description_text" => [],
+            "duration_certainty" => "KNOWN",
+            "effect" => "NO_SERVICE",
+            "effect_detail" => "STATION_CLOSURE",
+            "header_text" => [],
+            "id" => "113791",
+            "image" => %{
+              "localized_image" => [
+                %{
+                  "url" => url,
+                  "media_type" => "image/png",
+                  "language" => "en"
+                },
+                %{
+                  "url" => "some_other_url",
+                  "media_type" => "image/png",
+                  "language" => "fr"
+                }
+              ]
+            },
+            "informed_entity" => [%{}],
+            "last_modified_timestamp" => 1_494_947_991,
+            "last_push_notification_timestamp" => 1_494_947_991,
+            "service_effect_text" => [],
+            "severity" => 3,
+            "short_header_text" => [],
+            "timeframe_text" => []
+          }
+        ]
+      }
+
+      assert [%Alert{image: ^url}] = parse_json(map)
+    end
+
+    test "still returns image url even if no language specified" do
+      url =
+        "https://dev-mbta.pantheonsite.io/sites/default/files/styles/max_2600x2600/public/media/2023-09/QuincyCenter-Braintree-EA.png"
+
+      map = %{
+        "timestamp" => "1496832813",
+        "alerts" => [
+          %{
+            "active_period" => [],
+            "alert_lifecycle" => "NEW",
+            "cause" => "UNKNOWN_CAUSE",
+            "created_timestamp" => 1_494_947_991,
+            "description_text" => [],
+            "duration_certainty" => "KNOWN",
+            "effect" => "NO_SERVICE",
+            "effect_detail" => "STATION_CLOSURE",
+            "header_text" => [],
+            "id" => "113791",
+            "image" => %{
+              "localized_image" => [
+                %{
+                  "url" => url,
+                  "media_type" => "image/png"
+                }
+              ]
+            },
+            "informed_entity" => [%{}],
+            "last_modified_timestamp" => 1_494_947_991,
+            "last_push_notification_timestamp" => 1_494_947_991,
+            "service_effect_text" => [],
+            "severity" => 3,
+            "short_header_text" => [],
+            "timeframe_text" => []
+          }
+        ]
+      }
```

Comment:
question: should we have a test that we prefer an `en` image over a generic image?

Reply:
followup: should we have a test that we prefer an image with no language over a non-`en` language?